src/cats/monad/maybe.cljc

```diff
@@ -32,6 +32,8 @@
       (maybe/just 1)
       ;; => #<Just [1]>
   "
+  #?(:cljs
+    (:require-macros [cats.monad.maybe :refer (maybe)]))
```

Comment:
I'm pretty unfamiliar with ClojureScript; is this the correct incantation to include the `maybe` macro in the generated JS code?
For instance, I'm a bit confused as to how the `unless` macro is availabe to the ClojureScript tests when it isn't explicitly required via `:require-macros`: https://github.com/funcool/cats/pull/179/files#diff-3db711a639d69722715c32d496d63348R214